packages/odyssey-react/src/components/Status/index.tsx

```diff
@@ -58,13 +60,10 @@ const Status: FunctionComponent<Props> = (props) => {
     styles[`${variant}Variant`],
   );
 
-  const labelClass = labelHidden ? styles.labelVisuallyHidden : styles.label;
-
   return (
     <div className={styles.status} role="status">
-      <span className={labelClass}>
-        {label}
-      </span>
+      { !labelHidden && <span className={styles.label}>{label}</span> }
+      { labelHidden && <ScreenReaderText>{label}</ScreenReaderText> }
```

Comment:
this type of API isn't ideal, what we should consider allowing the label property to simply be more than just text such as `ReactNode` in which case a user can provide a screen reader hidden label or visible text